### ISSUE TITLE
More than one launch result appears on ReportPortal, when few specs specified in protractor config file

### ISSUE THREAD

**Issue by izabielin** [reportportal/agent-js-jasmine#4]
When we specify more then one spec (or suite) in protractor config file and trigger test run, on ReportPortal appears the same count of result launches as number of specs

**Comment by izabielin** [comment]
is any updates regarding current issue?

**Comment by lexecon** [comment]
As discussed: will create an example later

**Comment by izabielin** [comment]
@lexecon I will be very grateful if any real estimates (week, month etc.) is provided.

**Comment by izabielin** [comment]
after discussion with @lexecon we figured out the reason, why more than one launches appear (when more than one spec is specified) - it because we set shardTestFiles = true for chrome browser capability.  unfortunately it is mean, other problem exists and right now not possible to get all spec results in one launch when we run tests in parallel and i'm going to create separate issue on this
